keanu-project/src/main/java/io/improbable/keanu/distributions/continuous/InverseGamma.java

```diff
@@ -11,45 +11,55 @@
 import io.improbable.keanu.tensor.dbl.DoubleTensor;
 import io.improbable.keanu.vertices.dbl.KeanuRandom;
 
+/**
+ * @see "Computer Generation of Statistical Distributions
+ * by Richard Saucier,
+ * ARL-TR-2168 March 2000,
+ * 5.1.4 page 16"
+ */
 public class InverseGamma implements ContinuousDistribution {
 
     private final DoubleTensor alpha;
-    private final DoubleTensor beta;
+    private final DoubleTensor scale;
```

Comment:
I think `alpha` and `beta` are universally accepted, and it would be weird to change one and not the other.

Defer to a data scientist though.

Reply:
Beta is a scale parameter, I wanted to make the distinction from the shape parameter alpha.